dlp_mpi/mpi.py

```diff
@@ -74,6 +87,7 @@
         )
         raise
 
+if not _mpi_available:
```

Comment:
Why this line? Is it for readability?

Reply:
Yes, I found it cleaner, with an if statement